src/bidiMapper/domains/context/CdpTarget.ts

```diff
@@ -132,6 +134,12 @@ export class CdpTarget {
    * Enables all the required CDP domains and unblocks the target.
    */
   async #unblock() {
+    // Check if the network domain is enabled globally.
+    const enabledNetwork =
```

Comment:
If the browsing context is OOPiF, there already can be browsing context - specific subscriptions, so it should be checked as well

Reply:
I think the change should not be related to this PR, if you look at the current implementation we already always subscribe.